python/F3DPythonBindings.cxx

```diff
@@ -14,14 +14,69 @@
 
 namespace py = pybind11;
 
+namespace PYBIND11_NAMESPACE
+{
+namespace detail
+{
+template<>
+class type_caster<f3d::point3_t>
+{
+public:
+  bool load(handle src, bool convert)
+  {
+    if (!isinstance<sequence>(src))
+      return false;
+    py::sequence l = reinterpret_borrow<sequence>(src);
+    if (l.size() != 3)
+      return false;
+
+    size_t i = 0;
+    for (auto it : l)
+      value[i++] = py::cast<double>(it);
+    return true;
+  }
+
+  static handle cast(const f3d::point3_t& src, return_value_policy, handle /* parent */)
+  {
+    return Py_BuildValue("fff", src[0], src[1], src[2]);
+  }
+
+  PYBIND11_TYPE_CASTER(f3d::point3_t, const_name("f3d.point3_t"));
+};
+
+template<>
+class type_caster<f3d::vector3_t>
+{
+public:
+  bool load(handle src, bool convert)
+  {
+    if (!isinstance<sequence>(src))
+      return false;
+    py::sequence l = reinterpret_borrow<sequence>(src);
+    if (l.size() != 3)
+      return false;
+
+    size_t i = 0;
+    for (auto it : l)
+      value[i++] = py::cast<double>(it);
+    return true;
+  }
+
+  static handle cast(const f3d::vector3_t& src, return_value_policy, handle /* parent */)
+  {
+    return Py_BuildValue("fff", src[0], src[1], src[2]);
+  }
```

Comment:
add brackets everywhere ?

Reply:
ah yes, I skipped them on the single line blocks, was in python mode I guess :)
Will fix to conform to prescribed non-compact style

Reply:
`InsertBraces: true` another one for `.clang-format`